src/backend/api/meals.js

```diff
@@ -66,6 +72,7 @@ router.delete("/:id", async (request, response) => {
     throw error;
   }
 });
+
 //query params
 router.get("/", async (request, response) => {
   const query = request.query;
```

Comment:
Good work in using knex to query the database and filter the meals directly instead of loading them and using the filter method.
One thing though that will not work, is multiple query parameters The only case you considered is if we pass in maxPrice and limit.
But for instance, if I wanted to see meals with a certain title AND also created after a date, I will only get results based on my title parameter!